runtime/src/main/scala/akka/grpc/javadsl/ServiceHandler.scala

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.grpc.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import scala.annotation.varargs
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.control.NonFatal
+
+import akka.http.javadsl.model.HttpRequest
+import akka.http.javadsl.model.HttpResponse
+import akka.http.javadsl.model.StatusCodes
+import akka.japi.{Function => JFunction}
```

Comment:
Did you find a reason not to use `java.util.function.Function` here?

Reply:
I'll change that occurrence also together with the other one

Reply:
I'll do that in this PR